src/lambdaconnect_sync/db_drivers/memory.cljc

```diff
@@ -563,27 +565,27 @@
       entities))
 
   (objects-by-uuids [this database entity uuids fetch-inverses?]
-    (let [index (get-in database [:snapshots
+    (let [index (or (get-in database [:snapshots
```

Comment:
The or here allows push to emit better error messages when the collection is empty and should not be empty.